library/VTKExtensions/Rendering/vtkF3DRenderer.cxx

```diff
@@ -695,16 +695,16 @@ void vtkF3DRenderer::ConfigureTextActors()
   double textColor[3];
   if (this->IsBackgroundDark())
   {
-    textColor[0] = textColor[1] = textColor[2] = 1.0;
+    textColor[0] = textColor[1] = textColor[2] = 0.9;
   }
   else
   {
-    textColor[0] = textColor[1] = textColor[2] = 0.0;
+    textColor[0] = textColor[1] = textColor[2] = 0.5;
```

Comment:
why 0.5 specifically ?

Reply:
As stated above, it would stand for medium gray, but I can change it to 0.1 so it's 90% black. Totally up to you!

Reply:
Well, I'm not sure, that is why I'm asking you, why did you choose 0.5 ? :)

Reply:
Aesthetic reasons only. I thought medium gray would look a bit better in contrast to pure white.

Reply:
Ok!